components/compliance-service/compliance.go

```diff
@@ -611,13 +617,6 @@ func Serve(conf config.Compliance, grpcBinding string) error {
 	if err != nil {
 		return err
 	}
-	defer func() {
```

Comment:
This should be there so that on closure of the service, the go workers should also terminate by itself.